docs/handbook/common-tasks.md

```diff
@@ -65,15 +65,47 @@ ITEMS TO WRITE ABOUT:
 </div>
 
 
-## Multimedia
+## Multimedia - Audio
 
-< place holder >
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **DOC TEAM NOTE:**
+<div class="well">
+ITEMS TO WRITE ABOUT:
+
+* How to use the hidden `gstreamer-properties` configuration utility.
+* Pulse Audio configuration
+* [possible resource](https://www.solarismultimedia.com/)
+* [How it works](http://tuxradar.com/content/how-it-works-linux-audio-explained)
+
+</div>
+
+OpenIndiana employs several different audio components which work together to facilitate audio output and control.
+These systems are, in a sense, layered and each provides specific functions, although not without a significant a degree of crossover.
+The reason for so many different systems is largely a case of ensuring support for legacy as well as modern applications.
+
+| System | Description
+| --- | ---
+| Gstreamer | Mixer/Proxy
+| PulseAudio | Mixer/Proxy
+| OSS/OSSv4 | Open Sound System
+| Sun Audio | Solaris Audio API (Dev Audio)
```

Comment:
This paragraph seems cumbersome. I don't understand all these bits deeply, but... AFAIK, Illumos Audio system provides native Sun Audio APIs. Later OSS and OSSv4 APIs were added. These are low level OS interfaces to manage audio devices. PulseAudio provides more high-level API to manage devices and mixers, provided by userland.   Gstreamer is a set of API to manage not only device access, but to perform management of streams (video/audio/etc) and encoding, so I wouldn't mix them. It would be perfect if you could ask Garrett to comment on this section, when it'll be in better state.